index.html

```diff
@@ -1,67 +1,219 @@
-        </a>
-        </nav>
-       </header>
+    <nav class="navbar">
+        <a href="#"><div class="ezra"><h4>Ezra</h4></div></a>
+        <a href="index3.html"><img src="image/hamburger.png" alt="Hamburger Icon" class="hamburger"></a>
+    </nav>
 
-       <section class="headline">
-        <h1 class="primary">Hey There. <br>I'm Ezra.</h1>
-       
-        <h3 class="supportive">I'm a Software Developer</h3>
+    <header class="header">
+        <div><h1 class="primary">Hey There. <br>I'm Ezra.</h1></div>
+        <div><h3 class="supportive">I'm a Software Developer</h3></div>
+        <div><p class="paragraph-header">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p></div>
+            <div>
+                <ul class="social">
+                    <li><a href="#"><img src="/image/github.png" alt="Github"></a></li>
+                    <li><a href="#"><img src="image/linkedin.png" alt="LinkedIn"></a></li>
+                    <li><a href="#"><img src="image/thumbs.png" alt="Thumbs"></a></li>
+                    <li"><a href="#"><img src="image/twitter.png" alt="Twitter"></a></li>
+                    <li><a href="#"></a></li>
+                </ul>
+            </div>
+            <div class="group-header"><img src="image/Group.png" alt="Header Group"></div>
+    </header>
 
-        <p class="paragraph">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-            don’t hesitate to contact me.</p>
+    <section class="main">
+        <div class="recent"><h2>My Recent Works</h2></div>
+        <div class="grid">
+            <div class="workswidth">
+                <div class="works1">
+                    <div class="yogalady"> <img src="image/yogalady.png" alt="Yogalady"></div>
+                    <div class="multi-post1"><h3 class="multi-post1">Multi-Post Stories</h3></div>
+                    <div class="para-works1"><p class="para-works1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p></div>
+                    <div class="list1">
+                        <ul class="boxes1">
+                            <li class="box box1"><img src="image/css1.png" alt="CSS"></li>
+                            <li class="box box2"><img src="image/html1.png" alt="HTML"></li>
+                            <li class="box box3"><img src="image/bootstrap1.png" alt="Bootstrap"></a></li>
+                            <li class="box box4"><img src="image/ruby1.png" alt="Ruby"></li>
+                        </ul>
+                    </div>
+                    <button class="button1">See Project</button>
```

Comment:
- [ ] you have not implemented the button interaction which is a requirement for this milestone as seen in the image below
![req-btn-int](https://user-images.githubusercontent.com/89055502/235330062-99c93858-2749-4a80-ad0f-88815a4956e1.png)


> - i will provide you with possible steps to achieving this
- first off you need to give every button a unique class name eg `btn` and then style this btn class using pseudo selctors such as `:hover. :active` and apply the various background color and color from the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=bLjL2tJJSSkEfkRj-0) to these styling.
- so this is what your css code would look like
```
.btn:hover {
  background-color: red;
  color: white
}
```
- [ ] Note the example below is only for hover and i have only used random color do  well to check the figma for the correct color as shown below
![btn-t-2-int](https://user-images.githubusercontent.com/89055502/235335073-f03f98e6-db34-4ef9-be10-9c7f547b0cbc.png)